### PR TITLE
node10 support & bug fixes

### PR DESCRIPTION
**Pull request by raghits**
### Description of the Change

- Removed support for node6 & python2
- Added support for node10
- Renamed template folder names (lambda -> function)
- Leverage CloudWatch log group tags instead of parsing log group names for identifying service details
- Dedicated search API for service dashboard
- Bugfix related to log destination policy updates

### CODE REVIEW

File: core/jazz_services/index.js [hunk 1]
```diff
@@ -125,6 +125,7 @@ module.exports.handler = (event, context, cb) => {
                     // fetch services list from dynamodb, filter if required
                     fetchServices: function (onComplete) {
                         var query = event.query;
+                        query['isSearch'] = true;
```
Comment (devsatishm) [inline core/jazz_services/index.js]: @raghits  @dmpl1  Shouldn't this be `true` only when the path is `/services/search`?

Comment (raghits) [reply core/jazz_services/index.js]: @devsatishm This is implement in https://github.com/tmobile/jazz/pull/1223

Comment (dmpl1) [reply core/jazz_services/index.js]: @devsatishm @raghits, path check for search is addressed separately in above PR [#1223], Please additional comments related to search  issue is also addressed at #1223.

File: builds/delete-serverless-service-build-pack/Jenkinsfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!groovy.
+#!mgroovy.
```
Comment (devsatishm) [inline builds/delete-serverless-service-build-pack/Jenkinsfile]: Spell err..

Comment (raghits) [reply builds/delete-serverless-service-build-pack/Jenkinsfile]: Fixed the typo. This really didn't break the build pack.

**Pull request merged**